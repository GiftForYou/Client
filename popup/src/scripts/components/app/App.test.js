import React from 'react';
import {App,mapStateToProps} from './App';
import renderer from 'react-test-renderer';

const state={doc:{
  doc_user:'',
  doc_detail:[''],
  doc_recomendation:[]
}}
// const component = renderer.create(<App ></App>);

test('[Popup Test] overAll ', () => {
  // let tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});
//
// test('[Popup Test] app type', () => {
//   let tree = component.toJSON();
//   expect(tree.props).toMatchSnapshot();
// });
//
// test('[Popup Test] app have style', () => {
//   let tree = component.toJSON();
//   expect(tree.props.style).toMatchSnapshot();
// });
//
// test('[Popup Test] style have height', () => {
//   let tree = component.toJSON();
//   expect(tree.props.style.height).toMatchSnapshot();
// });
// test('[Popup Test] style have width', () => {
//   let tree = component.toJSON();
//   expect(tree.props.style.width).toMatchSnapshot();
// });
//
// test('[Popup Test] app have children', () => {
//   let tree = component.toJSON();
//   expect(tree.children).toMatchSnapshot();
// });

// test('[Popup Test] app mapStateToProps empty', () => {
//   expect(mapStateToProps(state)).toMatchSnapshot();
// });
//
// test('[Popup Test] app mapStateToProps doc_user', () => {
//   state.doc.doc_user='TEST';
//   expect(mapStateToProps(state)).toMatchSnapshot();
// });
//
// test('[Popup Test] app mapStateToProps doc_detail', () => {
//   state.doc.doc_detail='TEST';
//   expect(mapStateToProps(state)).toMatchSnapshot();
// });
//
// test('[Popup Test] app mapStateToProps doc_recomendation', () => {
//   state.doc.doc_recomendation='TEST';
//   expect(mapStateToProps(state)).toMatchSnapshot();
// });
