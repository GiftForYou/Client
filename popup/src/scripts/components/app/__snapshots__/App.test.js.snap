// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Popup Test] app have children 1`] = `
Array [
  <p>
    <button
        onClick={[Function]}
    >
        Analize
    </button>
    <button
        onClick={[Function]}
    >
        Show Recomendation
    </button>
</p>,
  <p />,
]
`;

exports[`[Popup Test] app have style 1`] = `
Object {
  "height": 600,
  "width": 800,
}
`;

exports[`[Popup Test] app mapStateToProps doc_detail 1`] = `
Object {
  "doc_detail": "TEST",
  "doc_recomendation": "",
  "doc_user": "TEST",
}
`;

exports[`[Popup Test] app mapStateToProps doc_recomendation 1`] = `
Object {
  "doc_detail": "TEST",
  "doc_recomendation": "TEST",
  "doc_user": "TEST",
}
`;

exports[`[Popup Test] app mapStateToProps doc_user 1`] = `
Object {
  "doc_detail": "",
  "doc_recomendation": "",
  "doc_user": "TEST",
}
`;

exports[`[Popup Test] app mapStateToProps empty 1`] = `
Object {
  "doc_detail": "",
  "doc_recomendation": "",
  "doc_user": "",
}
`;

exports[`[Popup Test] app type 1`] = `
Object {
  "style": Object {
    "height": 600,
    "width": 800,
  },
}
`;

exports[`[Popup Test] overAll  1`] = `
<div
  style={
    Object {
      "height": 600,
      "width": 800,
    }
  }
>
  <p>
    <button
      onClick={[Function]}
    >
      Analize
    </button>
    <button
      onClick={[Function]}
    >
      Show Recomendation
    </button>
  </p>
  <p />
</div>
`;

exports[`[Popup Test] style have height 1`] = `600`;

exports[`[Popup Test] style have width 1`] = `800`;
